src-plugins/itkFilters/itkFiltersWindowingProcess_p.h

```diff
@@ -61,10 +59,7 @@ class itkFiltersWindowingProcessPrivate : public itkFiltersProcessBasePrivate
         output->setData ( windowingFilter->GetOutput() );
         
         //Set output description metadata
-        QString newSeriesDescription = input->metadata ( medMetaDataKeys::SeriesDescription.key() );
-        newSeriesDescription += " intensity filter";
-        
-        output->addMetaData ( medMetaDataKeys::SeriesDescription.key(), newSeriesDescription );
+        medUtilities::setDerivedMetaData(output, input, "intensity");
```

Comment:
I would put the new intensity range of the image.
Something like "windowed [min max]"